app/controllers/platform/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+module Platform
+  class ApplicationController < ::ApplicationController
+    include Authentication
+  end
+end
```

Comment:
I am not sure this controller add any value if we can include the module on both controller really 🤔